### PR TITLE
AHTI-193 | Multiple calls to MyHelsinki API using different query parameters

### PR DESCRIPTION
**Pull request by charn**
Adds location+distance based filter to MyHelsinki mapping config

Refs AHTI-193

### CODE REVIEW

File: features/importers/myhelsinki_places/tests/conftest.py [hunk 1]
```diff
@@ -13,6 +13,8 @@ def autouse_db(db):
 
 @pytest.fixture
 def importer():
+    mhi = MyHelsinkiImporter()
+    mhi.api_calls = [{}]  # Response is mocked, parameters are redundant
     return MyHelsinkiImporter()
```
Comment (ahie) [inline features/importers/myhelsinki_places/tests/conftest.py]: Should return `mhi`?

File: features/importers/myhelsinki_places/importer.py [hunk 2]
```diff
@@ -61,6 +61,14 @@ class MyHelsinkiImporter(FeatureImporterBase):
 
     source_system = "myhelsinki"
     source_type = "place"
+
+    # Query parameters: http://open-api.myhelsinki.fi/doc#/v1places/listAll
+    api_calls = [
```
Comment (ahie) [inline features/importers/myhelsinki_places/importer.py]: Optional: could these be moved to a separate configuration file? Under the `myhelsinki_places` folder perhaps. Would possibly make usage of the importer more clear.

File: features/importers/myhelsinki_places/tests/conftest.py (hunk 1 above)
Comment (charn) [reply features/importers/myhelsinki_places/tests/conftest.py]: It should. :sweat_smile:

File: features/importers/myhelsinki_places/importer.py (hunk 2 above)
Comment (charn) [reply features/importers/myhelsinki_places/importer.py]: This might make sense. Though I think I could do a separate PR for that.

Comment (ahie) [reply features/importers/myhelsinki_places/importer.py]: Separate PR for this is fine as it could also include the unrelated change for `tag_config`.

**Pull request merged**